kin-recovery/src/main/java/kin/recovery/backup/view/CreatePasswordFragment.java

```diff
@@ -42,6 +43,7 @@ public static CreatePasswordFragment newInstance(@NonNull final BackupNavigator
 	private BackupNavigator nextStepListener;
 	private KeyboardHandler keyboardHandler;
 	private CreatePasswordPresenter createPasswordPresenter;
+	private KinAccount kinAccount; //TODO don't like the idea that it is here although no one use it because we only passing it to the presenter, any suggestions?
```

Comment:
well you can say it for any other module we're using here like keyboardHandler, I'm ok with it as long we're just gluing it together here

Reply:
ok